### DISCUSSION TITLE
How can ap values be visualized during VOC dataset training and testing

### DISCUSSION THREAD

**Discussion by gaobo25** [open-mmlab/mmdetection#8979] [Q&A]
### What is the problem this feature will solve?

How can ap values be visualized during VOC dataset training and testing

### What is the feature you are proposing to solve the problem?

How can ap values be visualized during VOC dataset training and testing

**Comment by ZwwWayne** [comment]
What kind of visualization do you want to do?
You can use tensorboard to log and visualize the curves of AP changed during training. You can also try tools/analysis_tools/analyze_logs.py

**Comment by gaobo25** [comment]
> What kind of visualization do you want to do? You can use tensorboard to log and visualize the curves of AP changed during training. You can also try tools/analysis_tools/analyze_logs.py

I want to save it as a picture. What should I do

**Comment by ZwwWayne** [comment]
They are already saved inthe log. You can use tools/analysis_tools/analyze_logs.py to draw them to a picture.

**Reply by gaobo25** [reply]
Well, thank you very much

**Reply by gaobo25** [reply]
How to output the ap ar value of each category after each step of training in log
